bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/pom.xml

```diff
@@ -36,7 +36,7 @@ limitations under the License.
 
   <properties>
     <mrunit.version>1.1.0</mrunit.version>
-    <google-cloud-bigtable-emulator.version>0.153.0</google-cloud-bigtable-emulator.version>
+    <google-cloud-bigtable-emulator.version>0.154.0</google-cloud-bigtable-emulator.version>
```

Comment:
Is there a reason for this dependency to be 'google-cloud-bigtable-emulator' while the other is 'bigtable-emulator'?

Reply:
good catch - I'll make them consistent!